composer.json

```diff
@@ -12,12 +12,13 @@
   ],
   "require": {
     "wpackagist-plugin/nginx-helper": "~1.9",
-    "wpackagist-plugin/woocommerce": "3.1.2"
+    "wpackagist-plugin/woocommerce": "3.1.2",
+    "vlucas/phpdotenv": "^2.5"
```

Comment:
If we add this here it will mean we always have to add composer.json without which our build won't pass.

Reply:
We use composer to install plugins in most of the projects and we need them in deployer.